app/decorators/ext_management_system_decorator.rb

```diff
@@ -6,6 +6,10 @@ def fonticon
   end
 
   def listicon_image
-    "100/vendor-#{image_name}.png"
+    if kind_of?(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      'vendor-hawkular'
+    else
+      "100/vendor-#{image_name}.png"
```

Comment:
Yes i know it is inconsistent, as the handling of those is inconsistent in list view vs quad icon. This is also why I am doing the indirection via listicon_image.

We could probably go over a lot of code path(s) with a fine comb and make it consistent that we always only return either of the two.

`middleware_topology_controller.js` is expecting the plain middle name.
`app/views/layouts/quadicon/_single_quad.html.haml` expects the full path
and so on